splash/resources.py

```diff
@@ -102,6 +105,7 @@ def _writeOutput(self, data, request, content_type=None):
             content_type = self.content_type
 
         if isinstance(data, (dict, list)):
+            data = bytes_to_unicode(data)
```

Comment:
What about fixing it with `splash.lua.lua2python` function in appropriate places? We should apply `splash.lua.lua2python` with care because Lua <-> Python converison is not loseless, but there shouldn't be any problems if we apply this function e.g. to the result of Lua's `main` function (if it is not applied yet).